dia.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
```

Comment:
Nice job! Your code compiled and ran! My only suggestion is to include comments!